src/content/es/2019/markup.md

```diff
@@ -0,0 +1,261 @@
+
+Se pueden observar dos cosas aquí. Primero, el conjunto de elementos que tienen más del 1% de uso no son exclusivamente HTML. De hecho, *27 de los 100 elementos más populares ni siquiera son HTML* - son SVG! Y hay *etiquetas no estándar en o muy cerca de ese límite también*!  Segundo, tenga en cuenta que menos del 1% de las páginas utilizan una gran cantidad de elementos HTML.
+
+Entonces, ¿todos esos elementos que son utilizados por menos del 1% de las páginas "inútiles"? Definitivamente no. Por eso es importante establecer una perspectiva. Hay alrededor [dos mil millones de sitios web en la web](https://www.websitehostingrating.com/internet-statistics-facts/). Si algo aparece en el 0.1% de todos los sitios web de nuestro conjunto de datos, podemos extrapolar que esto representa quizás *dos milliones de sitios web* en toda la web. Incluso 0.01% se extrapola a _doscientos mil sitios_.  Esta es también la razón por la cual eliminar el soporte para elementos, incluso aquellos muy antiguos que creemos que no son buenas ideas, es algo muy raro. Romper cientos de miles o millones de sitios simplemente no es algo que los proveedores de navegadores puedan hacer a la ligera.
+
+Muchos elementos, incluso los nativos, aparecen en menos del 1% de las páginas y siguen siendo muy importantes y exitosos. `<code>`, por ejemplo, es un elemento que uso y encuentro mucho. It's definitely useful and important, and yet it is used on only 0.57% of these pages.  Part of this is skewed based on what we are measuring; las páginas de inicio son generalmente *menos probable* para incluir ciertos tipos de cosas (como `<code>` por ejemplo). Las páginas de inicio tienen un propósito menos general que, por ejemplo, encabezados, párrafos, enlaces y listas. Sin embargo, los datos son generalmente útiles.
+
+También recopilamos información sobre qué páginas contenían un autor definido (no nativo) `.shadowRoot`. Alrededor del 0.22% de las páginas de escritorio y el 0.15% de las páginas móviles tenían un _shadow root_. Esto puede no parecer mucho, pero es más o menos 6.500 sitios en el conjunto de datos móviles y 10.000 sitios en el escritorio y es más que varios elementos HTML. `<summary>` por ejemplo, tiene un uso equivalente en el escritorio y es el elemento número 146 más popular. `<datalist>` aparece en el 0.04% de las páginas de inicio y es el elemento 201 más popular.
+
+De hecho, más del 15% de los elementos que contamos según lo definido por HTML están fuera de los 200 primeros en el conjunto de datos de escritorio. `<meter>` es el elemento menos popular de la "era HTML5", que podemos definir como 2004-2011, antes de que HTML se moviera a un modelo de Living Standard. Es alrededor del elemento número 1.000 en popularidad. `<slot>`, el elemento introducido más recientemente (abril de 2016) es solo alrededor del 1.400 elemento más popular.
```

Comment:
Rewrite last chunk to avoid saying like 1400th equivalent in Spanish

```suggestion
De hecho, más del 15% de los elementos que contamos según lo definido por HTML están fuera de los 200 primeros en el conjunto de datos de escritorio. `<meter>` es el elemento menos popular de la "era HTML5", que podemos definir como 2004-2011, antes de que HTML se moviera a un modelo de Living Standard. Es alrededor del elemento número 1.000 en popularidad. `<slot>`, el elemento introducido más recientemente (abril de 2016) está situado alrededor del puesto 1.400 en cuanto a popularidad.
```